.github/workflows/starter.yml

```diff
@@ -0,0 +1,15 @@
+name: Workflow starter
+
+on: [push, pull_request]
+
+jobs:
+    CodeQL:
```

Comment:
IMHO the CodeQL workflow lasts too long to be the first step. The first check should be just a single very simple build.

Reply:
Done, it lasts under 1 minute.